docs/excel/custom-functions-requirements.md

```diff
@@ -0,0 +1,30 @@
+---
+ms.date: 01/30/2019
+description: Learn how to use custom functions on different platforms. 
+title: Custom functions requirements (preview)
+localization_priority: Priority
+---
+# Custom functions requirements
+
+Custom functions are currently available in developer preview on the following platforms:
+
+- Excel Online
+- Excel for Windows (64-bit version 1810 or later). At present, Excel for Windows 32-bit may not work for all scenarios. 
+- Excel for Mac (version 13.329 or later)
+
+## Excel Online
+To use custom functions within Excel Online, login by using either your Office 365 subscription or a [Microsoft account](https://account.microsoft.com/account). 
+
+## Excel for Windows and Excel for Mac
+To use custom functions within Excel for Windows or Excel for Mac, you must have an Office 365 subscription, join the [Office Insider](https://products.office.com/office-insider) program (**Insider** level -- formerly called "Insider Fast"), and use a sufficiently recent build of Excel (as specified earlier in this note).
+
+If you are using a version of Office on your desktop which you downloaded from the Windows Store, you must be part of the [Windows Insider](https://insider.windows.com/) program at the **Insider** level (formerly called "Insider Fast"), running the April 2018 Update version or later to use custom functions. This is a new change as of January 2019.
+
+## Subscribe to Office 365
+If you don't already have an Office 365 subscription, you can get one by joining the [Office 365 Developer Program](https://developer.microsoft.com/en-us/office/dev-program).
+
+
+* [Custom functions overview](custom-functions-overview.md)
```

Comment:
Future idea. Eventually lets come back and add a next step which goes off to the quick start instead of all these links.

Reply:
Good idea!